curriculum/structure/superblocks/full-stack-developer.json

```diff
@@ -719,6 +719,7 @@
           "comingSoon": true,
           "blocks": [
             "lecture-classes-and-objects",
+            "lab-planet-class",
             "workshop-musical-instrument-inventory",
```

Comment:
```suggestion
            "workshop-musical-instrument-inventory",
            "lab-planet-class",
```

The order is different from the sheet:

<img width="744" height="342" alt="image" src="https://github.com/user-attachments/assets/1904b1a3-3e16-4182-947b-7f6d7a1386c1" />